certified-connectors/Alkymi/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1271 @@
+        },
+        "data": {
+          "type": "object"
+        }
+      },
+      "required": [
+        "uuid",
+        "created",
+        "document_uuid",
+        "document",
+        "dataset",
+        "pipeline",
+        "status",
+        "data"
+      ]
+    },
+    "Schema": {
+      "title": "Schema",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "maxLength": 50,
+          "pattern": "^[-a-zA-Z0-9_]+$"
+        },
+        "primary": {
+          "type": "boolean"
+        },
+        "fields": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SchemaField"
+          }
+        }
+      },
+      "required": [
+        "name",
+        "primary",
+        "fields"
+      ]
+    },
+    "SchemaField": {
+      "title": "SchemaField",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "maxLength": 50,
+          "pattern": "^[-a-zA-Z0-9_]+$"
+        },
+        "field_type": {
+          "type": "string"
+        },
+        "sub_schema": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "name",
+        "field_type"
+      ]
+    },
+    "WebHookCreatedResponse": {
+      "description": "WebHookCreatedResponse",
+      "properties": {
+        "uuid": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "uuid"
+      ],
+      "title": "WebHookCreatedResponse",
+      "type": "object"
+    },
+    "WebHookEvent": {
+      "title": "WebhookEvent",
+      "type": "object",
+      "properties": {
+        "uuid": {
+          "type": "string",
+          "description": "uuid"
+        },
+        "status": {
+          "type": "string",
+          "description": "status"
+        },
+        "created": {
+          "type": "string",
+          "description": "created"
+        },
+        "records": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "records"
+        },
+        "archived": {
+          "type": "boolean",
+          "description": "archived"
+        },
+        "reviewed": {
+          "type": "boolean",
+          "description": "reviewed"
+        },
+        "documents": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "documents"
+        },
+        "file_name": {
+          "type": "string",
+          "description": "file_name"
+        },
+        "pipeline_records": {
+          "type": "object",
+          "description": "pipeline_records"
+        },
+        "trigger_event_list": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "trigger_event_list"
+        }
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "datasets",
+      "description": ""
```

Comment:
@tori-alkymi Please provide a valid description value for the property.

Applies the same comment for all the similar properties in the swagger.